### PR TITLE
SAMV71Q21 PAC + VSCode utilities

### PR DESCRIPTION
**Pull request by SteelPh0enix**
- Added SAMV71Q21 PAC crate, generated via `svd2rust`
- Added VSCode tasks and scripts for building Aerugo and checking code with Clippy
- Added VSCode `launch.json` template for debugging on SAMV71Q21 board

### CODE REVIEW

File: arch/cortex-m/samv71q21-pac/build.rs
Comment (Glamhoth) [inline arch/cortex-m/samv71q21-pac/build.rs:1]: PAC itself isn't being build to any binary, should this be here?

File: scripts/check_code.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+aerugo_x86() {
+    cargo clippy -p aerugo -F use-aerugo-x86 -- -D warnings
```
Comment (Glamhoth) [inline scripts/check_code.sh:4]: add `--tests` too maybe?

**Review by Glamhoth — CHANGES_REQUESTED** [review]

File: arch/cortex-m/samv71q21-pac/build.rs
Comment (SteelPh0enix) [reply arch/cortex-m/samv71q21-pac/build.rs:1]: This build script is copying `device.x` from PAC crate to build root, which allows it to be seen by linker and attached to final binary.

I think that's a good place for this, an alternative would be adding `device.x` to every single example that would use PAC crate, along with inclusion of copying this linker script in example's `build.rs`, so it saves few copy&pastes for every example.

Comment (Glamhoth) [reply arch/cortex-m/samv71q21-pac/build.rs:1]: ack

**Review by Glamhoth — APPROVED** [review]

**Pull request merged**